### PR TITLE
Add `button` to mute audio in `Penguin Mark` demo

### PR DESCRIPTION
**Pull request by alrra**
Fix MicrosoftEdge/Demos#210

### CODE REVIEW

**Comment by sarvaje** [comment]
@alrra I don't see any code to "unmute" the audio.
IMO if you can mute, you should be able to 'unmute' the audio

**Comment by alrra** [comment]
> I don't see any code to "unmute" the audio.

@sarvaje As opposed to #256, this demo had only the mute option,  and since this seems like one of the performance demos, I thought that I should keep the changes to the bare minimum.

> IMO if you can mute, you should be able to 'unmute' the audio

Just say so, and I'll add that.

**Comment by sarvaje** [comment]
Ok, then LGTM :)

**Review by sarvaje — APPROVED** [review]

File: penguinmark/Default.html [hunk 1]
```diff
@@ -62,6 +62,7 @@
                 <button id="ResetButton" class="btn">Try Again</button>
             </div>
         </div>
+        <button id="MuteAudioButton" class="btn" autocomplete="off">Mute audio</button>
```
Comment (alrra) [inline penguinmark/Default.html]: `autocomplete="off"` is for Firefox which, after reloading the page, keeps a button disabled with JavaScript, disabled (see: https://bugzilla.mozilla.org/show_bug.cgi?id=654072).

**Comment by molant** [comment]
I'll add an un mute option. We are already messing with the audio, might as well do it right for the user.

**Comment by melanierichards** [comment]
@alrra ping once you've added the "unmute" button, thanks!

**Comment by melanierichards** [comment]
This PR is still in progress, right @alrra? (re: un-mute button)

**Comment by alrra** [comment]
> This PR is still in progress, right @alrra? (re: un-mute button)

@melanierichards Yes (got caught up with some of the other things).

**Comment by alrra** [comment]
> ping once you've added the "unmute" button, thanks!

ping @melanierichards 

---

Note: @molant asked to make the mute/unmute button disappear once `Test your browser` button is clicked, but retain the audio state.

**Comment by alrra** [comment]
Note: @molant asked to make the mute/unmute button disappear once `Test your browser` button is clicked.

**Comment by melanierichards** [comment]
@alrra woo hoo! You are finally free of the penguins.

**Pull request merged**